doc/source/rllib-algorithms.rst

```diff
@@ -364,3 +364,17 @@ Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rll
    :language: python
    :start-after: __sphinx_doc_begin__
    :end-before: __sphinx_doc_end__
+
+Alpha Zero (contrib/AZ)
+-------------------------------------------------
+
+`[paper] <https://arxiv.org/abs/1712.01815>`__ `[implementation] <https://github.com/ray-project/ray/blob/master/rllib/contrib/az>`__ AlphaZero is a powerful RL agent originally designed to solve 2 players games. This version adapts it to handle single player games. The code can be used with the SyncSampleOtimizer as well as with a modified version of the SyncReplayOptimizer and scales to any number of workers. It also implements the ranked rewards `(R2) <https://arxiv.org/abs/1807.01672>`__ strategy to enables self-play even in the one player setting. The code is mainly purposed to be used for combinatorial optimization.
+
+Tuned examples: `CartPole-v0 <https://github.com/ray-project/ray/blob/master/rllib/contrib/az/examples/train_cartpole.py>`__
```

Comment:
Do you have another benchmark available yet? It would be great to have both a fast toy example and a larger scale one.

Reply:
We don't have for now tested on a publicly available env with large scale training. We only experimented on company envs that we can not unfortunately share. But we are developing one player combinatorial game envs that we will release in the next months once we got satisfying baseline results.